### PR TITLE
Show list posts and status

### PR DESCRIPTION
**Pull request by methanol-dev**
## Related Tickets:
- [#47116](https://edu-redmine.sun-asterisk.vn/issues/47116)

## WHAT this PR does?
- Show list posts and status

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by methanol-dev** [comment]
ready

File: resources/views/user/post/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                @endphp
+                @foreach ($posts as $post)
+                <tr>
+                    <th scope="row">{{ ++$i }}</th>
+                    <td>{{ $post->title }}</td>
+                    <td>
+                        @php
+                            if($post->status == 1){
+                                echo 'PENDING';
+                            }elseif($post->status == 2){
```
Comment (ghost) [inline resources/views/user/post/index.blade.php]: bỏ status vào config

File: resources/views/user/post/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                @php
+                    $i = 0;
+                @endphp
+                @foreach ($posts as $post)
+                <tr>
+                    <th scope="row">{{ ++$i }}</th>
+                    <td>{{ $post->title }}</td>
+                    <td>
+                        @php
+                            if($post->status == 1){
```
Comment (ghost) [inline resources/views/user/post/index.blade.php]: dùng @switch case

**Comment by methanol-dev** [comment]
ready

File: app/Http/Controllers/PostController.php [hunk 3]
```diff
@@ -13,6 +13,14 @@
 
 class PostController extends Controller
 {
+    public function index()
+    {
+        $user_id = Auth::user()->id;
```
Comment (huyenttm-0901) [inline app/Http/Controllers/PostController.php]:
```suggestion
        $user_id = Auth::id();
```

File: app/Http/Controllers/PostController.php [hunk 4]
```diff
@@ -13,6 +13,14 @@
...
+        $posts = Post::where('user_id', $user_id)->paginate(config('constants.pagination'));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/PostController.php]: dùng relationship

**Comment by methanol-dev** [comment]
ready

File: app/Http/Controllers/PostController.php [hunk 5]
```diff
@@ -13,6 +14,14 @@
 
 class PostController extends Controller
 {
+    public function index()
+    {
+        $user_id = Auth::id();
+        $posts = User::find($user_id)->posts()->paginate(config('constants.pagination'));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/PostController.php:20]:
```suggestion
        $posts = Auth::user()->posts()->paginate(config('constants.pagination'));
```

File: resources/views/user/post/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                @php
+                $i = 0;
+                @endphp
+                @foreach ($posts as $post)
+                <tr>
+                    <th scope="row">{{ ++$i }}</th>
+                    <td>{{ $post->title }}</td>
+                    <td>
+                        @switch($post->status)
+                            @case(config('constants.pending'))
```
Comment (huyenttm-0901) [inline resources/views/user/post/index.blade.php]:
```suggestion
                        @switch ($post->status)
                            @case (config('constants.pending'))
```

**Comment by methanol-dev** [comment]
ready

**Pull request merged**